Sources/AppcuesKit/Presentation/Actions/ActionRegistry.swift

```diff
@@ -35,17 +35,12 @@ internal class ActionRegistry {
         register(action: AppcuesRequestReviewAction.self)
     }
 
-    func register(action: AppcuesExperienceAction.Type) {
-        guard actions[action.type] == nil else {
-            #if DEBUG
-            if ProcessInfo.processInfo.environment["XCTestBundlePath"] == nil {
-                assertionFailure("Action of type \(action.type) is already registered.")
-            }
-            #endif
-            return
-        }
+    @discardableResult
+    func register(action: AppcuesExperienceAction.Type) -> Bool {
+        guard actions[action.type] == nil else { return false }
```

Comment:
not critical really, now that it is not public, but we're losing the reason why `false` is returned I guess. Maybe just a comment in the api doc in Appcues.swift could say "will not be registered if the given action type is already registered".

Reply:
Good point. I can log an error here easily enough so I'll add that back in along with the comment.